pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const list = [];
+    const { error, isLoading, product } = useProductInfo(productId, list);
+    const { description, is_visible: isVisible, name, price, type } = product ?? {};   
+    const typeCapitalized = type?.replace(/^\w/, (c: string) => c.toUpperCase());
+    const isVisibleString = isVisible?.toString().replace(/^\w/, (c: string) => c.toUpperCase());
+
+    if (isLoading) return <Loading />;
+    if (error) return <ErrorMessage error={error} />;
+  
+    return (
+        <>
+            <Panel header="Basic Information" margin="xxSmall">
```

Comment:
I believe the issues with margin will be solved with the bug fix we looked at earlier... if we want, we can probably just override the bottom margin here, setting the first margin to `small` and then `none` for the second

```suggestion
            <Panel header="Basic Information" marginBottom="small">...</Panel>
            <Panel header="Description" margin="none">...</Panel>
```